src/redux/covid/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const baseUrl = 'https://disease.sh/v3/covid-19/countries';
+
+const fetchData = async () => {
+  const stats = [];
+  const response = await axios.get(baseUrl);
+  const responseStat = response.data;
+
+  responseStat.map(({ countryInfo: { _id: id, flag }, ...data }) => {
```

Comment:
[ Optional Changes ] 
This change is not mandatory in the requirements and won't stop your project from being approved but I highly recommend you consider working on it for your future projects. 

- [ ] Your app looks great and works fine, but I highly suggest when using Redux efficiently, you use the [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) hook provided by redux for performing async actions. Kindly use that next time when working with redux and fetching API data. 👍🏽